### PR TITLE
login: google login with click

### PR DESCRIPTION
**Pull request by matallo**
prevent google auto login when in `/login` and google_sign_in is true

CR @Kartones @xavijam

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2896/))

File: app/views/google_plus/_google_plus_button.html.erb [hunk 1]
```diff
@@ -8,14 +8,40 @@
...
-    if(typeof authResult.access_token != 'undefined') {
+    if (typeof authResult.access_token != 'undefined') {
       var access_token_field = document.getElementById('<%= config.access_token_field_id %>');
       var google_signup_access_token = document.getElementById('google_signup_access_token');
       // INFO: user exists, let's sign in
       if (authResult['status']['signed_in'] && access_token_field.value === '' && google_signup_access_token.value === '') {
         access_token_field.value = authResult.access_token;
-        access_token_field.form.submit();
+
+        var access_token_filled = document.getElementById('google_access_filled');
```
Comment (Kartones) [inline app/views/google_plus/_google_plus_button.html.erb]: why not moving this to just below function definition? dcode is duplicated and IIRC JS variable hoisting will exactly do that, move all `var` to beggining of the function

**Comment by Kartones** [comment]
looks ok :+1:

**Comment by iriberri** [comment]
Oh please merge. I have angry clients :(

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2911/))

**Pull request merged**